internal/provider/provider.go

```diff
@@ -201,12 +207,15 @@ func providerConfigure(st state.StateI) (func(ctx context.Context, d *schema.Res
 		}
 		apiClient.twin_id = twin
 		var cl rmb.Client
-		if apiClient.use_rmb_proxy {
-			verify_reply := d.Get("verify_reply").(bool)
-			cl, err = client.NewProxyBus(rmb_proxy_url, apiClient.twin_id, apiClient.substrateConn, identity, verify_reply)
-		} else {
-			cl, err = rmb.NewRMBClient(apiClient.rmb_redis_url)
+
+		relayURL, ok := RelayURLs[network]
```

Comment:
user should be able to override the relayurl, should be added to the schema above